runtime/etc/src/mli/chpl-mli-server-runtime.c

```diff
@@ -139,7 +141,9 @@ void chpl_mli_smain(const char* setup_conn) {
 
   chpl_mli_debugf("Shutdown, code: %s\n", chpl_mli_errstr(id));
 
+#ifdef CHPL_MLI_DEBUG_PRINT
   seconds = (double) (clock() - before) / (double) CLOCKS_PER_SEC;
+#endif
 
   chpl_mli_close(chpl_server.setup_sock);
   chpl_mli_close(chpl_server.main);
```

Comment:
It seems potentially confusing to future developers to only wrap these two references to `seconds` and not the call to `chpl_mli_debugf`.  Should we add a comment saying why it isn't protected in the same way?